packages/cedar/src/Chart.ts

```diff
@@ -67,7 +74,8 @@ export interface IDefinition {
   type?: string
   specification?: {}
   overrides?: {},
-  legend?: ILegend
+  legend?: ILegend,
+  styles?: IStyles
```

Comment:
I had planned to call this `style` but I think you're right to call it `styles` so it doesn't get confused w/ CSS.